test/testOffline.js

```diff
@@ -257,6 +257,31 @@ describe('Offline', function() {
     });
   });
 
+  it('should cache directoryIndexes as dirs', function() {
+    var rootDir = temp.mkdirSync('oghliner');
+    var dir = path.join(rootDir, 'dist');
+    fs.mkdirSync(dir);
+    var subDir = path.join(dir, 'subdir');
+    fs.mkdirSync(subDir);
+
+    fs.writeFileSync(path.join(subDir, 'test_file_1.js'), 'test_file_1');
+    fs.writeFileSync(path.join(subDir, 'test_file_2.js'), 'test_file_2');
+    fs.writeFileSync(path.join(subDir, 'test_file_3.js'), 'test_file_3');
+
+    process.chdir(rootDir);
+
+    return offline({
+      rootDir: dir,
+      directoryIndexes: ['test_file_1.js']
+    }).then(function() {
+      var content = fs.readFileSync(path.join(dir, 'offline-worker.js'), 'utf8');
+      assert.notEqual(content.indexOf('/'), -1, 'Must have a directory in the output');
```

Comment:
I would still change this line to look for the index of the text `'./'` (including the single quotes) to make it a more reliable assertion about the presence of a reference to the root directory.